cleanlab/datalab/datalab.py

```diff
@@ -188,6 +189,13 @@ def find_issues(
             To best detect label issues, provide this input obtained from the most accurate model you can produce.
 
             If provided, this must be a 2D array with shape (num_examples, K) where K is the number of classes in the dataset.
+            Must not be provided if `predictions` is provided.
```

Comment:
```suggestion
```

Reply:
philosphy-wise, let's focus the current documentation mostly around `pred_probs` still being the primary input to Datalab. 

So keep the explanations of when to use `predictions` vs. pred_probs contained within the docstring of the `predictions` argument.

Reply:
The safest approach here is to have a unified argument at the user-level. We'll keep `pred_probs` for all tasks, but don't have to modify the signatures of low-level classes like `RegressionLabelIssueManager`.

What these things have in common is that these are model-outputs that we use as inputs to Datalab. For  `pred_probs` and `predictions` in particular, they are just prediction-based model outputs.
I've added helper classes for handling each type of model output, but that won't change the top-level API for the user that use `Datalab.find_issues(pred_probs=...)`.
It's backwards compatible, easily extensible to other types of tasks and the other argument types like `features`, even `knn_graph`.

These helper classes are just an intermediate representation that won't get passed on to the issue managers directly, so this requires minimal changes.

Reply:
This means I've removed the docstring for the `predictions` argument, but clarified the expected layout/structure of `pred_probs` in the relevant classes. That's much easier to maintain.